rasa/nlu/classifiers/keyword_intent_classifier.py

```diff
@@ -1,32 +1,94 @@
-from typing import Any, Optional, Text
+import os
+import logging
+import re
+import typing
+from typing import Any, Dict, Optional, Text
 
+from rasa.nlu import utils
 from rasa.nlu.components import Component
 from rasa.nlu.training_data import Message
 
+logger = logging.getLogger(__name__)
+
+if typing.TYPE_CHECKING:
+    from rasa.nlu.config import RasaNLUModelConfig
+    from rasa.nlu.training_data import TrainingData
+    from rasa.nlu.model import Metadata
+    from rasa.nlu.training_data import Message
+
 
 class KeywordIntentClassifier(Component):
+    """Intent classifier using simple keyword matching.
+
+
+    The classifier takes a list of keywords and associated intents as an input.
+    A input sentence is checked for the keywords and the intent is returned.
+
+    """
 
     provides = ["intent"]
 
-    his = ["hello", "hi", "hey"]
+    intent_keyword_map = {}
 
-    byes = ["bye", "goodbye"]
+    def train(
+        self,
+        training_data: "TrainingData",
+        cfg: Optional["RasaNLUModelConfig"] = None,
+        **kwargs: Any
+    ) -> None:
+        self.intent_keyword_map = {}
+        for intent in training_data.intents:
+            self.intent_keyword_map[intent] = [
+                ex.text
```

Comment:
I'm confused, so we don't have any special format?

Reply:
No, we had a discussion about it in slack (https://rasa-hq.slack.com/archives/C36SS4N8M/p1560960162110400) and the final decision was that classifiers should never be combined and therefore the keywords for an intent will be exactly the examples for that intent.